.devcontainer/example-welcome-message.txt

```diff
@@ -0,0 +1,3 @@
+👋 Welcome to Astro!
+
+The example app should automatically start soon in a new terminal tab.
```

Comment:
```suggestion
👋 Welcome to "Astro" in GitHub Codespaces!

🛠️  Your environment is fully setup with all the required software.

🚀 The example app should automatically start soon in a new terminal tab.
```

Reply:
Probably we could make similar changes to other welcome messages as well?

Reply:
Not following what you mean here.

Reply:
I am suggesting an update to the [example-welcome-message.txt](https://github.com/withastro/astro/pull/6288/files/7c4fce28e22b47fa85cca54600ac01d7a0737dc3#diff-c26282470ea314cfa922b5047b78b10fe214470a8284be71122f17c0ecab2c88) contents. If you decide to apply those changes, then you could also update other welcome files as well. (eg. devcontainer/welcome-message.txt)